### PR TITLE
css

### PR DESCRIPTION
**Pull request by RominaFior**

### CODE REVIEW

File: staff/RominaFior/css-layout-simple-adaptive/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import "mobilemain.css" screen and (max-width: 768px);
+@import "desktopmain.css" screen and (min-width:769px)
```
Comment (agandia9) [inline staff/RominaFior/css-layout-simple-adaptive/styles/main.css:2]: There are common CSS properties in both versions (desktop and mobile), these should go here to be applied always.

File: staff/RominaFior/css-layout-simple-adaptive/styles/desktopmain.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
+*{
+    margin: 0%;
+    padding: 0%;
+    box-sizing: border-box;
+    
+}
+html, body{
+    background: black;
+    color: white;
+    width: 100%;
```
Comment (manuelbarzi) [inline staff/RominaFior/css-layout-simple-adaptive/styles/desktopmain.css]: TIP is ```width: 100%```rule required?

File: staff/RominaFior/css-layout-simple-adaptive/styles/desktopmain.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    height:60% ;
+    border: 1px solid white;
+    
+}
+section{
+    width: 60%;
+    height: 100%;
+    float: left;
+    border: 1px solid white;
```
Comment (manuelbarzi) [inline staff/RominaFior/css-layout-simple-adaptive/styles/desktopmain.css]: TIP group selectors that share same rules

SEE Grouping Selectors https://www.w3schools.com/css/css_syntax.asp

File: staff/RominaFior/css-layout-simple-adaptive/styles/mobilemain.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
+*{
```
Comment (manuelbarzi) [inline staff/RominaFior/css-layout-simple-adaptive/styles/mobilemain.css]: TIP mobilemain.css file name is not very clear. just mobile.css in enough. rename it.

File: staff/RominaFior/css-layout-simple-adaptive/styles/desktopmain.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
+*{
```
Comment (manuelbarzi) [inline staff/RominaFior/css-layout-simple-adaptive/styles/desktopmain.css]: TIP desktopmain.css file name is not very clear. desktop.css is enough. rename it.

File: staff/RominaFior/css-layout-simple-adaptive/styles/main.css (hunk 1 above)
Comment (RominaFior) [reply staff/RominaFior/css-layout-simple-adaptive/styles/main.css:2]: Thanks for the tip 👍

File: staff/RominaFior/js/challenges/count-words/scripts/main.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var count= countWords('Hello world')
+
+console.log('countWords ("hello world" should return 2) ', count === 2, count);
+
+
+try {
+    convert = toCamelCase(true)
```
Comment (albanelia0) [inline staff/RominaFior/js/challenges/count-words/scripts/main.js]: Shouldn't 'convert' be declared like this `var convert` before the first try/catch? 🤔

Comment (RominaFior) [reply staff/RominaFior/js/challenges/count-words/scripts/main.js]: indeed! thank you Alba, now it shoud work!

File: staff/RominaFior/app-template/styles/register.css [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+.google:hover{
+    border-color:  #2dbe60;
+}
+
+ul {
+    list-style: none;
+}
+
+ul li:before {
+    content: '✓ ';
```
Comment (agandia9) [inline staff/RominaFior/app-template/styles/register.css:54]: cool! ;D

**Review by agandia9 — COMMENTED** [review]
okey!

**Pull request merged**